contracts/SSVNetwork.sol

```diff
@@ -632,16 +640,23 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
         emit ExecuteOperatorFeePeriodUpdate(newExecuteOperatorFeePeriod);
     }
 
+    function updateLiquidationThresholdPeriod(uint64 blocks) external onlyOwner override {
+        if(blocks < MINIMAL_LIQUIDATION_THRESHOLD) {
+            revert BelowMinimumBlockPeriod();
+        }
```

Comment:
This is coming from V2 right?  
Is MINIMAL_LIQUIDATION_THRESHOLD = 50 also coming from V2, mainly the actual value of 50?

Reply:
no, from initial test v3 logic